OJP_JourneySupport.xsd

```diff
@@ -82,7 +82,7 @@
 			</xs:element>
 			<xs:element name="ProductCategory" type="ProductCategoryStructure">
 				<xs:annotation>
-					<xs:documentation>A product category for the service.</xs:documentation>
+					<xs:documentation>A product category for the service. This is a classification that exists in NeTEx/Siri that can be used to identify groups of journeys with some special properties for marketing and fare products. E.g. "TE2" for SNCF or a special panorama train "PE" in Switzerland. </xs:documentation>
```

Comment:
This is a classification defined in NeTEx/SIRI used to identify groups of journeys with some special properties for marketing and fare products, e.g. "TE2" for SNCF or a special panorama train "PE" in Switzerland.

Reply:
changed